account-kit/java/signer-sdk/src/main/java/com/alchemy/aa/core/TekManager.java

```diff
@@ -1,66 +1,54 @@
 package com.alchemy.aa.core;
 
-import com.alchemy.aa.core.exceptions.NoTEKException;
+import com.alchemy.aa.core.exceptions.NoTekException;
+import com.google.crypto.tink.CleartextKeysetHandle;
 import com.google.crypto.tink.InsecureSecretKeyAccess;
 import com.google.crypto.tink.KeyTemplate;
 import com.google.crypto.tink.KeysetHandle;
 import com.google.crypto.tink.TinkJsonProtoKeysetFormat;
 import com.google.crypto.tink.hybrid.HpkeParameters;
 import com.google.crypto.tink.hybrid.HpkePrivateKey;
 import com.google.crypto.tink.hybrid.HpkePublicKey;
-import com.google.crypto.tink.CleartextKeysetHandle;
 import com.google.crypto.tink.hybrid.internal.HpkeContext;
 import com.google.crypto.tink.hybrid.internal.HpkeKemKeyFactory;
 import com.google.crypto.tink.hybrid.internal.HpkePrimitiveFactory;
-
 import com.google.crypto.tink.proto.KeyData;
 import com.google.crypto.tink.proto.Keyset;
 import com.google.crypto.tink.util.Bytes;
 import com.google.crypto.tink.util.SecretBytes;
 import com.google.protobuf.InvalidProtocolBufferException;
 import java.security.GeneralSecurityException;
 
-public class TEKManager {
-
-    public TEKManager() throws GeneralSecurityException {
-    }
+public class TekManager {
 
-    public static TEKManager InitializeTEKManagerFromHpkeKey(HpkePrivateKey privateKey)
-            throws GeneralSecurityException, InvalidProtocolBufferException {
+    public static TekManager fromHpkeKey(HpkePrivateKey privaTekey)
+            throws GeneralSecurityException {
         KeysetHandle keysetHandle = KeysetHandle.newBuilder()
-                .addEntry(KeysetHandle.importKey(privateKey).makePrimary().withFixedId(0)).build();
-        return InitializeTEKManagerFromKeySetHandle(keysetHandle);
+                .addEntry(KeysetHandle.importKey(privaTekey).makePrimary().withFixedId(0)).build();
+        return fromKeyetHandle(keysetHandle);
     }
 
-    public static TEKManager InitializeTEKManagerFromKeySetHandle(KeysetHandle keysetHandle)
+    public static TekManager fromKeyetHandle(KeysetHandle keysetHandle)
             throws GeneralSecurityException {
-        TEKManager tek = new TEKManager();
+        TekManager Tek = new TekManager();
```

Comment:
(Nit) Java convention uses lowercased variable names. Naming it `Tek` makes it sound like the name of a class and makes `Tek.serializedKeyset` below look like a static call.

Reply:
done